terra/Makefile

```diff
@@ -24,6 +24,7 @@ endif
 
 $(WASMS) artifacts/checksums.txt: $(SOURCE_FILES)
 	DOCKER_BUILDKIT=1 docker build --target artifacts -o artifacts .
+	sed -i 's/interface_version_7/interface_version_8/g' $(WASMS)
```

Comment:
just to remind ourselves, this will not be necessary

Reply:
we tried that approach and it was broken in subtle ways, so the current plan is to do option 2. My comment above is meant as a reminder to revert this change, as we're no longer doing it that way